### PR TITLE
Feature/atr 68 master bql outlining option to collapse all

### PR DESCRIPTION
**Pull request by SENya1990**
@k-s-popova please review changed and new strings in resources. There is one new option - to allow collapsing of BQL parts. Also I have split settings into two categories: "BQL Coloring" and "BQL Outlining"

### CODE REVIEW

File: src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx [hunk 1]
```diff
@@ -226,4 +228,10 @@
   <data name="Setting_UseBqlOutlining_Description" xml:space="preserve">
     <value>Collapse parts of BQL commands (only for Roslyn coloring).</value>
```
Comment (k-s-popova) [inline src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx]: Collapse BQL commands (only for Roslyn coloring).

File: src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx [hunk 2]
```diff
@@ -226,4 +228,10 @@
...
   </data>
+  <data name="Setting_UseBqlDetailedOutlining_Title" xml:space="preserve">
+    <value>Outline parts of the BQL</value>
```
Comment (k-s-popova) [inline src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx]: Outline parts of BQL commands

File: src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx [hunk 3]
```diff
@@ -226,4 +228,10 @@
...
+  </data>
+  <data name="Setting_UseBqlDetailedOutlining_Description" xml:space="preserve">
+    <value>Allow to collapse parts of the BQL (only for Roslyn coloring). If this setting is set to false the Acuminator will collapse only entire BQL queries.</value>
+  </data>
```
Comment (k-s-popova) [inline src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx]: Collapse parts of BQL commands (only for Roslyn coloring).  If set to false, BQL commands are collapsed only entirely.

File: src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx (hunk 2 above)
Comment (SENya1990) [reply src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx]: @k-s-popova Thanks)

**Pull request merged**